### ISSUE TITLE
Clarification that Models Trained Without Pre-trained Weights Can Be Used for Commercial Purposes

### ISSUE THREAD

**Issue by minato-ellie** [Deci-AI/super-gradients#1190]
### 📚 Documentation Improvement

To address the concerns raised in Issues #1174 , #983 , and #894 , I recommend explicitly stating in the documentation that **the trained model can be used for commercial purposes when not trained with pre-trained weights**. 
This statement will clarify the question of commercial viability of the model and alleviate related concerns.

Thank you for considering these suggestions

**Comment by BloodAxe** [comment]
The license text was approved by legal dept so I don't think we are changing.

**Comment by N950** [comment]
@BloodAxe Is Yolo NAS-Pose source code also Apache-2.0 (no pretrained weights just using the code to train from scratch) ?
Does all the super-gradients source code fall under Apache-2.0 ?

**Comment by BloodAxe** [comment]
Please search around for similar issues, this question was answered already.

**Comment by N950** [comment]
@BloodAxe I'm sorry if this seems repetitive or annoying, I've read all the issues related to licensing before asking and couldn't find explicit references to Yolo NAS-Pose.
I'm asking specifically about NAS-Pose because it has [it's own license](https://github.com/Deci-AI/super-gradients/blob/master/LICENSE.YOLONAS-POSE.md) and the wording of the license's text doesn't exclude the source code from `any components comprising the model and the associated documentation (the "Software")`.
